src/vocab_tree/vocab_tree.cpp

```diff
@@ -42,36 +42,38 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "vocab_tree.h"
 #include "clustering.h"
 
+#include <Eigen/Dense>
+
 using std::cout;
 using std::endl;
 
-float l2sq(DTYPE *a, DTYPE *b, int dim)
```

Comment:
I didn't configure sse or other advanced compile options in cmake. The comparison is done in debug mode. So i guess it's related to platforms.